desktop-themes/TraditionalOk/index.theme.in

```diff
@@ -1,12 +1,12 @@
 [Desktop Entry]
 Type=X-GNOME-Metatheme
-Name=TraditionalOk
-Comment=Port of Clearlooks-Phenix
+Name=Traditional-Ok
+Comment=Port of Clearlooks-Phenix and Traditional OK
 Encoding=UTF-8
 
 [X-GNOME-Metatheme]
-GtkTheme=TraditionalOk
-MetacityTheme=TraditionalOk
+GtkTheme=Traditional-Ok
+MetacityTheme=Traditional-Ok
```

Comment:
It will keep failing due to the mismatched name - revert this file and it should install correctly.